requirements/base.in

```diff
@@ -6,7 +6,7 @@ boto
 Django>=1.11
 django-model-utils
 edx-django-oauth2-provider
-djangorestframework==3.7.7
+djangorestframework==3.9.4
```

Comment:
Can we move this version pin to `requirements/constraints.txt`?  That would let us upgrade DRF again in edx-platform without needing to do another edxval release first (since `setup.py` is only parsing the version constraints listed directly in this file).

Reply:
I just moved this one to `constraints.txt`. Hopefully that's all I need to do.